scratchapi.js

```diff
@@ -7,9 +7,10 @@ var fs = require('fs');
 var WebSocket = require('ws');
 
 var SERVER = 'scratch.mit.edu';
-var PROJECTS_SERVER = 'projects.scratch.mit.edu';
-var CDN_SERVER = 'cdn.scratch.mit.edu';
-var CLOUD_SERVER = 'clouddata.scratch.mit.edu';
+var PROJECTS_SERVER = 'projects.' + SERVER;
+var CDN_SERVER = 'cdn.' + SERVER ;
+var CLOUD_SERVER = 'clouddata.' + SERVER;
+var API_SERVER = 'api.' + SERVER;
```

Comment:
I would prefer these to stay as fully qualified domain names. It's easier to copy and paste.

Reply:
Ok. Done.